echoregions/regions2d/regions2d.py

```diff
@@ -551,6 +552,9 @@ def transect_mask(
         bbox_distance_threshold: float
             The maximum value for how far apart the left and right bounding box for each transect
             value region. Default is set to 1 minute.
+        must_pass_check : bool
+            To determine whether fail if transect sequence check strings exist or pass and print
+            the transect sequence checks.
```

Comment:
This docstring is a little hard to decipher. Could you re-write it to have something in the form of
```suggestion
        must_pass_check : bool
            If True: ... 
            If False: ...
```

Reply:
Made this change.